lib/fog/storage/openstack/requests/delete_container.rb

```diff
@@ -1,6 +1,12 @@
 module Fog
   module Storage
     class OpenStack
+      class Mock
+        def delete_container(name)
+          fail "Mock Not Implemented (#delete_container) in: #{__FILE__}:#{__LINE__}"
```

Comment:
Actually, I added this myself to see where was the exception from. Fog only showed sometinhg like `Not yet implemented` and I wanted to get the exact location of the not implemented thing.